frontend/src/component/layouts/TabbedLayout.scss

```diff
@@ -0,0 +1,22 @@
+@import '../styles/font';
+
+.tab-layout {
+  display: flex;
+  height: 100%;
+  width: 100%;
+
+  .tab-headers {
+    flex: 0 0;
+  }
```

Comment:
I think we should remove this and leave tab content creators to decide whatever they want to do.